reactioncommerce/reaction
# fix: update sharp to match files

Signed-off-by: Akarshit Wal <akarshitwal@gmail.com>

Resolves #6417 
Impact: **critical**
Type: **bugfix**

## Issue
Sharp's version mismatch between api-plugin-files & reaction was causing this issue.

## Solution
Released a new version of api-plugin-files with the correct version & upgraded the sharp version in package.json.

## Breaking changes

None

## Testing
1. Create a local image with `docker build -t reactioncommerce/reaction:version-up .`
2. Run the image with `docker run --env-file ./.env -p 3000:3000 --network reaction.localhost -it reactioncommerce/reaction:version-up`
3. There should be no error.
